### PR TITLE
Goodbye classic mods

### PR DESCRIPTION
**Pull request by Mailaender**
The community around http://cnc-comm.com/ is [hostile](http://i.imgur.com/zPZDSVG.jpg). No one cared for the classic mods. They are also redundant and waste developer time, mostly @pchote during his refactors to add Tiberian Sun support.
- Created https://github.com/OpenRA/OpenRA/wiki/FAQ#this-is-not-true-to-the-original which will redirect them to the more popular http://cncnet.org/ and http://redalert1.com site.
- Will send deletion request for http://www.moddb.com/mods/openra-classic
- Just closed our TODO list at #2740 and will do the same with all related feature requests.

You can still send a patch if you want to maintain it again. Probably also remove all the redundancy in the yaml rules and inherit more from the ra/cnc mods to succeed.

### CODE REVIEW

**Comment by SMIFFGIG** [comment]
To be fair, if OpenRA did replicate the classics (or at least have a mode for it). Then there would be absolutely no point in bothering with the originals. It would also allow modders to do anything they liked.

It is a great shame that this is no longer being worked on. I hope someone does take this up again. I have no intent on installing all the old games again but would love to replay the classics as they were but on a fresh updated engine.

Maybe someone will pick this up again in the future.

**Comment by penev92** [comment]
TLDR it had became unmaintainable, and at the same time there was no pleasing the "true fans" (read "original C&C fanatics"), who could always find something to complain about.

**Comment by SMIFFGIG** [comment]
Thanks for the TLDR.

The community seems to be growing as OpenRA gets more popular. I don't know that for sure but it seems that way.
Maybe another developer or two will jump onboard in the future and kickstart this again. 
That's what I'm holding out for (also it seems important to have a classic mode for the missions to function properly) so maybe that will help when those are done...

**Comment by Nyerguds** [comment]
@SMIFFGIG The main problem is that from the very start of the OpenRA project, there has never been any research into how the C&C games actually worked internally.They never _tried_ to replicate the originals; they just made their own game engine and used the C&C games' tech trees and graphics for it.

Because of this, they simply don't feel the same in terms of how weapons, projectiles, movement, inaccuracy etc impact gameplay. This isn't something that can really be fixed; the game engines are just too different from the ground up.

**Comment by SMIFFGIG** [comment]
Hmm makes sense, I'm guessing it would need a bit of reverse engineering perhaps to get some kind of base values to work from if that is even possible?
I would like to see it, im sure it wouldn't be straightforward but could certainly go to emulate it very well.

If this is the case then I'm assuming the TS mod will be the same, as this is based off of OpenRA values which themselves are not correct?
Would it not make sense to as closely emulate TD/RA values to the originals before pressing on to far with the TS mod?
As TS was afterall based directly on the Red Alert 1 engine.

**Comment by GraionDilach** [comment]
I'll be honest. @SMIFFGIG if you cannot add anything to a discussion then don't necro it. I'd expect better from a TS veteran modder than necroing closed PRs every second day.

Featurewise the TD and the RA mod are 99% complete. They are pretty much done. That's what the OpenRA playerbase got used to and that's what they expect. 

The gameplay can be emulated via converting the rules but beyond rules not so much. Logics do differ and in many cases I dare stating OpenRA does a better job over WW. And honestly, it's not a big deal. This PR was merged because of no interest over the classic mods, and that haven't changed since. If you haven't realized it, it's _exactly your kind of people who demanded for reverse-engineered values and gameplay were behind the reason the classic mods were axed in the first place_. If TS gets tweaked during implementation - which is already happening because, well, just because Westwood was stupid, we don't have to be - then get over it. If you don't like it, then you can still
* play the originals
* write/maintain classic mods on your own
* ignore OpenRA

but please, stop resurrecting closed issues/PRs _without any meaning_. You're wasting everyone's time with this. If you can't resist doing that then go back to PPM where this behaviour is even encouraged. Thank you.

**Comment by SMIFFGIG** [comment]
Why the hostility Graion?
I am very interested in OpenRA and how it progresses.

I comment on PR's and issues WITH meaning,
And I am quite aware I can play the originals, etc etc

Get a grip, there is no need for you to be unnecessarily hostile.
Nyerguds and penev92 answered my questions and contributed to the discussion. The only comment that hasn't added anything to the above discussion is yours. Except for unwarranted hostility.

**Comment by penev92** [comment]
Let's keep discussions like this to the forums or to our IRC channel.
I too feel that GitHub is not the right place, as a lot of people get spammed emails for every comment. Because of that, and the nature of comments an issue or a PR suggests, things that are not work-related should not be here IMO.

Also hostility is never good, but @GraionDilach makes a good point for our mods.

**Comment by Mailaender** [comment]
This may be a good use/test case for https://github.com/OpenRA/OpenRA/issues/8993.

**Comment by Phrohdoh** [comment]
I don't see why RA2 can't fulfill that role as-is.
And @cjshmyr's Sole Survivor, etc.
